ember-app/app/mixins/subscriptions/card.js

```diff
@@ -54,6 +54,22 @@ var CardSubscriptionMixin = Ember.Mixin.create({
       });
     },
     labeled: sortedQueue(function(message) {
+      if(message.label){ //PR's do not have label arrays
+        return this.get("issue.other_labels").addObject(message.label);
+      }
+      this.set("issue.other_labels", message.issue.other_labels);
+    }, {time: 5000, sort: function(a,b){
+      var timeA = Date.parse(a.issue.updated_at);
+      var timeB = Date.parse(b.issue.updated_at);
+      return timeA - timeB;
+    }}),
+    unlabeled: sortedQueue(function(message) {
+      if(message.label){ //PR's do not have label arrays
+        var match = this.get("issue.other_labels").find((label)=>{
+          return label.name === message.label.name;
+        });
+        return this.get("issue.other_labels").removeObject(match);
+      }
```

Comment:
Add the same find check to the above `labeled` event to prevent adding the same label twice

We need to guard against things being out of sync